lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -5167,6 +5167,7 @@ DefMacroI('\verbatim@font', undef,
   '\fontfamily{\ttdefault}\fontseries{\mddefault}\fontshape{\updefault}\selectfont');
 
 Let('\reset@font', '\normalfont');
+DefMacro('\@fontswitch{}{}', '\ifmmode #2\relax\else #1 \fi');
```

Comment:
I snuck this `\@fontswitch` macro from latex.ltx in since raw interpretation stumbled on it, seemed simple/harmless enough.